api/overview/azure/data-lake-analytics.md

```diff
@@ -37,47 +37,37 @@ dotnet add package Microsoft.Azure.Management.DataLake.Analytics
 
 ### Example
 
---Example overview--
+This example creates the clients to connect with and manage the analytics account.
 
 ```csharp
-/* Code goes here */
-```
-
-> [!div class="nextstepaction"]
-> [Explore the client APIs](/dotnet/api/overview/azure/datalakeanalytics/client)
-
-
-## Management library
-
---Data Lake Analytics management blurb here--
-
-Install the [NuGet package](https://www.nuget.org/packages/Microsoft.Azure.Management.DataLake.Analytics) directly from the Visual Studio [Package Manager console][PackageManager] or with the [.NET Core CLI][DotNetCLI].
-
-#### Visual Studio Package Manager
+/*
+using AdlClient 
+*/
 
-```powershell
-Install-Package Microsoft.Azure.Management.DataLake.Analytics
-```
+/// Setup authentication for this demo
+Authentication auth = new Authentication("microsoft.onmicrosoft.com"); // change this to YOUR tenant
+auth.Authenticate();
 
-```bash
-dotnet add package Microsoft.Azure.Management.DataLake.Analytics
-```
-
-### Example
+// Collect info about the Azure resources needed for this demo
+string subid = "<Subscription Id>";
+string rg = "<Resource Group>";
+string adla_name = "<user Name>";
 
---Example overview--
+// Identify the accounts
+AnalyticsAccountRef adla_account = new AnalyticsAccountRef(subid, rg, adla_name);
 

```

Comment:
Change to: 

`AnalyticsAccountRef adla_account = new AnalyticsAccountRef(subscriptionId, resourceGroup, userName);`